### DISCUSSION TITLE
Process Original Images

### DISCUSSION THREAD

**Discussion by nikkoch** [FriendsOfShopware/FroshPlatformThumbnailProcessor#60] [Q&A] [answered]
Hello and thank you for the solution! 

We have a question here. What means the 'Process Original Images' checkbox in 'ThumbnailProcessor incl. Lazy Loading and Webp-Support' Shpoware 6 extension configuration? 

Have a good time!

**Comment by nikkoch** [comment]
Maybe this could be connected to duplicated product images?

**Comment by tinect** [comment, marked as answer]
This means thumbnail-pattern will also be passed to the original-image.
Original images are used for the product gallery or when no thumbnail has been setup or generated f.e.
These are then handled as "thumbnail" with a max size of 3000px, which size is meant as placeholder :-)

**Reply by nikkoch** [reply]
thank you tinect,
so original images should not add to load, if we switch it ON?
We have a lot of images loading now: https://ma*********ximex.shop/Abtropfgestell-Geschirr-mit-Abtropfschale-Schwarz/87290500-m

We are improving our load score, as you can suppose :)

**Reply by nikkoch** [reply]
for mobile surely

**Reply by tinect** [reply]
I don't know if I got your point. Sure, images are loading...
![image](https://user-images.githubusercontent.com/135993/178344330-b0ddc3c9-293c-480a-ae41-5ab8db1037a2.png)
This looks correct, first image usage in page of 1427586_Product.jpg needs 400px, after that the dom requests 800px. So all correct regarding thumbnail-loading. But I'd recommended adding thumbnail-size of 430px onto the album because the image preview just needs this size.

But you missed to setup any service for the thumbnails? You can find some examples [here](https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessor/discussions/categories/patterns).
And: remove the height-parameter as it is superfluous.

**Reply by nikkoch** [reply]
thank you for the advice tinect. 

We have set thedomain at cloudimage. Can we now just use this pattern in ThumbnailProcessor extension settings: https://token.cloudimg.io/v7/www.example.com/{mediaPath}?w={width}&func=bound&org_if_sml=1

Or should we also integrate cloudimage js?

**Reply by tinect** [reply]
I assume you just tried it in the meanwhile. 
But for the archive: Just set up the pattern. :-) there is no need for a JS library.

**Reply by nikkoch** [reply]
Many thanks for your kind assistance tinect!
